DataFormats/BTauReco/interface/DeepDoubleBTagInfo.h

```diff
@@ -1,15 +0,0 @@
-#ifndef DataFormats_BTauReco_DeepDoubleBTagInfo_h
-#define DataFormats_BTauReco_DeepDoubleBTagInfo_h
-
-#include "DataFormats/BTauReco/interface/FeaturesTagInfo.h"
-#include "DataFormats/BTauReco/interface/DeepDoubleBFeatures.h"
-
-namespace reco {
-
-typedef  FeaturesTagInfo<btagbtvdeep::DeepDoubleBFeatures> DeepDoubleBTagInfo;
```

Comment:
May I repeat my previous comment:


Is it obvious that we never wrote any useful data with ```DeepDoubleBTagInfo``` ?

To be clear: if some data was written and it will need to be read with the code including this PR, then this file should be kept at least in a dummy form ```struct DeepDoubleBTagInfo {};``` to avoid problems reading data with unknown input types.

Reply:
@slava77 can you elaborate on how to do this properly? Alternatively, could just keep the file?

Reply:
@andrzejnovak @jmduarte 
please clarify first on the original question: did we write any useful data with ```DeepDoubleBTagInfo``` ?
This is not a part of the standard miniAOD outputs, but may be in some samples used for development/analysis, which may need to be read in the release after this PR is merged.
My "preferred" answer is that we do not have any useful data with ```DeepDoubleBTagInfo``` and we can drop this file. However, this may be different from the real needs.

Reply:
I can't confidently exclude it. @jmduarte

Reply:
Given that DeepDoubleB is not run in the standard reco but only in the MiniAOD production, all these TagInfo have always been used as transient objects. This is at least my understanding.

Reply:
perhaps it's a good reason then to abandon the attempts to put compatibility dictionaries
and simply cleanup the old DeepDoubleB

Reply:
Well let me know then what should be done. I'm happy to push the struct changes or remove the double b references

Reply:
in the past two weeks, since https://github.com/cms-sw/cmssw/pull/24918#discussion_r229218138, there was no evidence presented of presence of files with this transient data saved.
So, it seems somewhat safe to simply cleanup all mention of the old type names.